browser/flagr-ui/src/main.js

```diff
@@ -12,6 +12,13 @@ import router from './router'
 Vue.config.productionTip = false
 Vue.use(ElementUI, { locale })
 
+// Autofocus certain fields
+Vue.directive('focus', {
+      inserted: function (el) {
+                el.__vue__.focus()
+            }
+})
+
```

Comment:
Please indent with 2 spaces, I may look into how to make the linter catch these in the future

Reply:
Updated